.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,10 +1,10 @@
-# NOTE(rfratto): this file *must* have the .yml extension otherwise GitHub doesn't 
+# NOTE(rfratto): this file *must* have the .yml extension otherwise GitHub doesn't
 # recognize it.
 blank_issues_enabled: true
 contact_links:
   - name: Grafana Agent community support
-    url: https://github.com/grafana/agent/discussions
-    about: If you need help or support, ask questions in Github Discussions.
+    url: https://community.grafana.com/c/support/grafana-agent
```

Comment:
It seems to be because "Grafana Agent" is a subcategory of "Grafana", but IMO it'd make sense to make Grafana Agent its own category. I'm just not sure if the old links that point to `/support/grafana-agent` will continue to work.